Kernel/Arch/aarch64/init.cpp

```diff
@@ -22,17 +22,17 @@
 #include <Kernel/Arch/aarch64/RPi/UART.h>
 #include <Kernel/Arch/aarch64/Registers.h>
 #include <Kernel/Arch/aarch64/TrapFrame.h>
-#include <Kernel/CommandLine.h>
+#include <Kernel/Boot/CommandLine.h>
 #include <Kernel/Devices/DeviceManagement.h>
 #include <Kernel/FileSystem/VirtualFileSystem.h>
 #include <Kernel/Graphics/Console/BootFramebufferConsole.h>
-#include <Kernel/JailManagement.h>
 #include <Kernel/KSyms.h>
+#include <Kernel/Library/Panic.h>
 #include <Kernel/Memory/MemoryManager.h>
-#include <Kernel/Panic.h>
-#include <Kernel/Scheduler.h>
+#include <Kernel/Security/JailManagement.h>
 #include <Kernel/Storage/StorageManagement.h>
```

Comment:
not sure whether `FileSystem` should go under `Storage`? or maybe even content of `FileSystem` moved into `Storage`? (did not read the code, inferring from the names only)

Reply:
I think it's OK as it is now :)